index.html

```diff
@@ -102,7 +102,7 @@ <h2 class="teaser-item__tittle">We are the heroes</h2>
        <img src="/images/newsletter/graduation_20191026MA-original.jpg" itemprop="image">
       </div>
     </article>
-    <div class="teaser-item teaser-item__img-first">
+    <div class="teaser-item teaser-item__img_img-first">
```

Comment:
The class you had before was fine and easier to remember. Let's make this `teaser-item_img-first. It's a modifier of `teaser-item`.